runtime/core/src/test/java/io/atlasmap/core/DefaultAtlasConversionServiceTest.java

```diff
@@ -678,24 +678,24 @@ public void testConvertTypeFromLong() throws AtlasConversionException {
 
     @Test
     public void testConvertTypeFromNumber() throws AtlasConversionException {
-        assertEquals(BigInteger.valueOf(1), service.convertType((Number)1, FieldType.NUMBER, FieldType.BIG_INTEGER));
-        assertEquals(true, service.convertType((Number)1, FieldType.NUMBER, FieldType.BOOLEAN));
-        assertEquals((byte)0x01, service.convertType((Number)1, FieldType.NUMBER, FieldType.BYTE));
-        assertEquals((char)1, service.convertType((Number)1, FieldType.NUMBER, FieldType.CHAR));
-        assertEquals(LocalDate.class, service.convertType((Number)1, FieldType.NUMBER, FieldType.DATE).getClass());
-        assertEquals(Date.class, service.convertType((Number)1, FieldType.NUMBER, FieldType.DATE_TIME).getClass());
-        assertEquals(ZonedDateTime.class, service.convertType((Number)1, FieldType.NUMBER, FieldType.DATE_TZ).getClass());
-        assertEquals(ZonedDateTime.class, service.convertType((Number)1, FieldType.NUMBER, FieldType.TIME_TZ).getClass());
-        assertEquals(ZonedDateTime.class, service.convertType((Number)1, FieldType.NUMBER, FieldType.DATE_TIME_TZ).getClass());
-        assertEquals(BigDecimal.valueOf(1), service.convertType((Number)1, FieldType.NUMBER, FieldType.DECIMAL));
-        assertEquals(1.0d, service.convertType((Number)1, FieldType.NUMBER, FieldType.DOUBLE));
-        assertEquals(1.0f, service.convertType((Number)1, FieldType.NUMBER, FieldType.FLOAT));
-        assertEquals(1, service.convertType((Number)1, FieldType.NUMBER, FieldType.INTEGER));
-        assertEquals(1L, service.convertType((Number)1, FieldType.NUMBER, FieldType.LONG));
-        assertEquals(1, service.convertType((Number)1, FieldType.NUMBER, FieldType.NUMBER));
-        assertEquals((short)1, service.convertType((Number)1, FieldType.NUMBER, FieldType.SHORT));
-        assertEquals("1", service.convertType((Number)1, FieldType.NUMBER, FieldType.STRING));
-        assertEquals(LocalTime.class, service.convertType((Number)1, FieldType.NUMBER, FieldType.TIME).getClass());
+        assertEquals(BigInteger.valueOf(1), service.convertType(1, FieldType.NUMBER, FieldType.BIG_INTEGER));
```

Comment:
Any reason removing casting to Number? IIRC without this cast the NumberConverter is not picked up, instead IntegerConverter is used.

Reply:
Sorry, that was done automatically by a Save As action in my IDE. I've changed it a bit so the IDE doesn't mess it up.